Example/Example-Info.plist

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think that the Example-Info.plist can be removed as it does not contain any additional information. You will have to remove it from the Xcode Project configuration and this the builds and archives fine without any problems on my machine.

Reply:
Removed in [c7d76ef](https://github.com/StanfordBDHG/ResearchKitOnFHIR/pull/59/commits/c7d76efcf6d5367a838f0df9a6559f719d71bf9a)